tests/responses/BaseResponse.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+
+namespace Bitbucket\Tests\responses;
+
+
+class BaseResponse
+{
+    /**
+     * @param $path
```

Comment:
invalid syntax

Reply:
I don't understand, how can I solve this?

Reply:
You are missing the type.